plasmapy/formulary/collisions.py

```diff
@@ -526,13 +685,13 @@ def impact_parameter(
         b_{max} = \left(\lambda_{De}^2 + a_i^2\right)^{1/2}
 
     The minimum impact parameter is typically some combination of the
-    thermal deBroglie wavelength and the distance of closest approach
+    thermal de Broglie wavelength and the distance of closest approach
     for a 90 degree Coulomb collision. A quadratic sum is used for
     all GMS methods, except for GMS-5, where b_min is simply set to
```

Comment:
```suggestion
    all GMS methods, except for GMS-5, where ``b_min`` is simply set to
```
Minor restructuredText formatting change.